packages/commonwealth/client/scripts/models/Account.ts

```diff
@@ -88,14 +88,23 @@ class Account {
       DISCOURAGED_NONREACTIVE_fetchProfilesByAddress(chain?.id, address).then(
         (res) => {
           const data = res[0];
-          updatedProfile.initialize(
-            data?.name,
-            data.address,
-            data?.avatarUrl,
-            data.id,
-            updatedProfile.chain,
-            data?.lastActive
-          );
+          if (!data) {
```

Comment:
Purpose for adding this new conditional? Seems unrelated.

Reply:
There was another console error here that I have been seeing occasionally. In cases where the address hasn't been used yet:

<img width="484" alt="Screenshot 2023-10-18 at 12 30 26 PM" src="https://github.com/hicommonwealth/commonwealth/assets/9438198/17ab98c8-47bd-4c0a-a308-09883c935a11">


It is unrelated, but worth cleaning up. Ok to fix separately if preferred.